prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -401,6 +401,37 @@ periodics:
       secret:
         secretName: google-oss-robot-ssh-keys
         defaultMode: 0400
+- cron: "05 15 * * 1"  # Run at 7:05 PST (15:05 UTC) Mon
+  name: ci-oss-test-infra-autobump-monitoring
+  cluster: test-infra-trusted
+  decorate: true
+  extra_refs:
+  - org: GoogleCloudPlatform
+    repo: oss-test-infra
+    base_ref: master
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+  annotations:
+    testgrid-dashboards: googleoss-test-infra
+    testgrid-tab-name: auto-update-monitoring
```

Comment:
nit: either name the tab autobump-monitoring or change the prowjob name to auto-update-monitoring to make it easier to understand which job the tab is for.

Reply:
done, thanks!